src/Compiler.Dynamic/SystemWebCompilation.cs

```diff
@@ -50,7 +50,7 @@ internal sealed class SystemWebCompilation : IDisposable, IWebFormsCompiler
     private SystemWebCompilationUnit CompileAllPages(ICompilationStrategy strategy, CancellationToken token)
     {
         var aspxFiles = Files.GetFiles().Where(t => t.FullPath.EndsWith(".aspx"))
-            .Select(t => new VirtualPath("/" + t.FullPath));
+            .Select(t => new VirtualPath("/" + t.FullPath.Replace("\\", "/")));
```

Comment:
Can you move this into the VirtualPath class to normalize all instances of a path?